src/Airspace/NearestAirspace.hpp

```diff
@@ -35,13 +35,13 @@ class ProtectedAirspaceWarningManager;
 class NearestAirspace {
 
 public:
-  const AbstractAirspace *airspace;
+  const AbstractAirspace *airspace = nullptr;
```

Comment:
Again an initialiser with an arbitrary value.

Reply:
Unnecessary. Instead it is initialized in the default constructor.